packages/xml-views-validation/test/validators/document/non-unique-id-spec.ts

```diff
@@ -0,0 +1,170 @@
+import { expect } from "chai";
+import { UI5SemanticModel } from "@ui5-language-assistant/semantic-model-types";
+import { generateModel } from "@ui5-language-assistant/test-utils";
+import { validateNonUniqueID } from "../../../src/validators/document/non-unique-id";
+import { NonUniqueIDIssue } from "../../../api";
+import {
+  computeExpectedRanges,
+  testValidationsScenario,
+} from "../../test-utils";
+
+describe("the use of non unique id validation", () => {
+  let ui5SemanticModel: UI5SemanticModel;
+
+  let testNonUniqueIDScenario: (opts: {
+    xmlText: string;
+    assertion: (issues: NonUniqueIDIssue[]) => void;
+  }) => void;
+
+  before(async () => {
+    ui5SemanticModel = await generateModel({ version: "1.74.0" });
+
+    testNonUniqueIDScenario = (opts): void =>
+      testValidationsScenario({
+        model: ui5SemanticModel,
+        validators: {
+          document: [validateNonUniqueID],
+        },
+        xmlText: opts.xmlText,
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        assertion: opts.assertion as any,
```

Comment:
Why do we need to use `any` here?

Reply:
![image](https://user-images.githubusercontent.com/4233843/83523738-47608880-a4eb-11ea-9ec0-1984e2996341.png)

Reply:
I think it can probably be solved by defining a generic type for `assertion` (that `extends UI5XMLViewIssue`)